mlflow/sagemaker/__init__.py

```diff
@@ -115,7 +116,9 @@ def build_image(name=DEFAULT_IMAGE_NAME, mlflow_home=None):
         if mlflow_home:
             mlflow_dir = _copy_project(
                 src_path=mlflow_home, dst_path=tmp.path())
-            install_mlflow = "COPY {mlflow_dir} /opt/mlflow\n RUN pip install /opt/mlflow\n"
+            install_mlflow = ("COPY {mlflow_dir} /opt/mlflow\n"
+                              "RUN cd /opt/mlflow/mlflow/java && mvn package\n"
```

Comment:
Let's cd `mlflow/java/scoring`, we don't need `mlflow/java/client`. We should probably also add `-DskipTests`.